hs_core/hydroshare/users.py

```diff
@@ -603,7 +612,11 @@ def get_resource_list(creator=None,
 
             if group:
                 group = group_from_id(group)
-                queries[t].append(Q(edit_groups=group) | Q(view_groups=group))
+                queries[t].append(Q(raccess__r2grp__privilege__lte=PrivilegeCodes.CHANGE,
+                                    raccess__r2grp__group=group.gaccess) |
+                                  Q(raccess__r2grp__privilege__lte=PrivilegeCodes.VIEW,
+                                    raccess__r2grp__group=group.gaccess)
+                                  )
```

Comment:
Same query as above: this should be what the access library provides, not something you need to do yourself.